### PR TITLE
Use GitHub actions

### PR DESCRIPTION
**Pull request by moonso**
This PR updates the GitHub Actions file so it should be possible to run tests

**Expected outcome**:
GitHub Actions should run tests
A new badge should show in README
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR

### CODE REVIEW

**Comment by moonso** [comment]
GitHub Actions is set up here as a first step. I can start to move things from travis to GitHub Actions in other PRs. This is ready to be reviewed

**Comment by northwestwitch** [comment]
I'll review this now!

File: scout/parse/ensembl.py [hunk 1]
```diff
@@ -262,7 +261,8 @@ def parse_ensembl_exons(lines):
 
         exon["start"] = start
         exon["end"] = end
-        exon["exon_id"] = "-".join([str(exon["chrom"]), str(start), str(end)])
```
Comment (northwestwitch) [inline scout/parse/ensembl.py]: 👍

File: scout/parse/variant/conservation.py [hunk 2]
```diff
@@ -16,6 +17,7 @@ def parse_conservations(variant, parsed_transcripts=[]):
         Returns:
             conservations(dict): A dictionary with the conservations
     """
+    parsed_transcripts = parsed_transcripts or []
```
Comment (northwestwitch) [inline scout/parse/variant/conservation.py]: 👍

File: scout/parse/variant/conservation.py [hunk 3]
```diff
@@ -82,20 +84,19 @@ def parse_conservation_csq(transcript, field_key):
...
-        if scores:
-            for score in scores.split(
-                "&"
-            ):  # fiels may consist of multiple numeric values
-                score = float(score)
-                if score >= CONSERVATION[field_key]["conserved_min"]:
-                    conservations.append("Conserved")
-                else:
-                    conservations.append("NotConserved")
+        if not scores:
```
Comment (northwestwitch) [inline scout/parse/variant/conservation.py]: 👍

File: scout/server/blueprints/cases/views.py [hunk 4]
```diff
@@ -561,7 +545,7 @@ def gene_variants(institute_id):
     data = {}
     if (form.hgnc_symbols.data) and len(form.hgnc_symbols.data) > 0:
         is_clinical = form.data.get("variant_type", "clinical") == "clinical"
-        clinical_symbols = store.clinical_symbols(case_obj) if is_clinical else None
+        # clinical_symbols = store.clinical_symbols(case_obj) if is_clinical else None
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/views.py]: What about removing the commented lines here?

File: scout/server/blueprints/cases/views.py [hunk 5]
```diff
@@ -571,8 +555,8 @@ def gene_variants(institute_id):
                     hgnc_symbols.append(hgnc_gene["hgnc_symbol"])
             elif store.hgnc_genes(hgnc_symbol).count() == 0:
                 not_found_symbols.append(hgnc_symbol)
-            elif is_clinical and (hgnc_symbol not in clinical_symbols):
-                non_clinical_symbols.append(hgnc_symbol)
+            # elif is_clinical and (hgnc_symbol not in clinical_symbols):
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/views.py]: And here

File: scout/server/blueprints/variant/verification_controllers.py [hunk 6]
```diff
@@ -63,7 +64,7 @@ def variant_verification(
     pp(variant_obj)
     recipients = institute_obj["sanger_recipients"]
     if len(recipients) == 0:
-        raise MissingSangerRecipientError()
+        raise MissingVerificationRecipientError()
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/verification_controllers.py]: Yes, of course!

File: scout/server/blueprints/variants/controllers.py [hunk 7]
```diff
@@ -1,56 +1,53 @@
...
+    MANUAL_RANK_OPTIONS,
+    MOSAICISM_OPTIONS,
+    SEVERE_SO_TERMS,
+    SPIDEX_HUMAN,
+    VERBS_MAP,
 )
 from scout.constants.acmg import ACMG_CRITERIA
 from scout.constants.variants_export import EXPORT_HEADER, VERIFIED_VARIANTS_HEADER
 from scout.export.variant import export_verified_variants
+from scout.server.blueprints.genes.controllers import gene
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/controllers.py]: These black changes are a mess to review!

File: scout/server/blueprints/variants/controllers.py [hunk 8]
```diff
@@ -417,6 +414,7 @@ def variant_export_lines(store, case_obj, variants_query):
             variant_line.append(";".join(str(x) for x in gene_names))
             variant_line.append(";".join(str(x) for x in hgvs_c))
         else:
+            i = 0
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/controllers.py]: Is this the only change to this file?

File: scout/server/blueprints/variants/views.py [hunk 9]
```diff
@@ -82,15 +83,15 @@ def variants(institute_id, case_name):
         file = request.files[form.symbol_file.name]
 
     if request.files and file and file.filename != "":
-        log.debug("Upload file request files: {0}".format(request.files.to_dict()))
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/views.py]: Wow this was so wrong!

File: scout/utils/scout_requests.py [hunk 10]
```diff
@@ -310,13 +314,13 @@ def fetch_hpo_files(genes_to_phenotype=False, phenotype_to_genes=False):
     hpo_urls = {}
 
     if genes_to_phenotype is True:
-        hpo_urls["genes_to_phenotype"] = base_url.format("genes_to_phenotype.txt")
```
Comment (northwestwitch) [inline scout/utils/scout_requests.py]: Wow here!

File: README.md [hunk 11]
```diff
@@ -6,6 +6,7 @@
 </p>
 
 [![Build Status][travis-img]][travis-url]
+![Build Status - GitHub](https://github.com/Clinical-Genomics/scout/workflows/Scout%20tests/badge.svg)
```
[the remark this answers is not in the document]
Comment (moonso) [reply README.md]: Yes It will work when run on master:)

File: scout/server/blueprints/variants/controllers.py (hunk 7 above)
Comment (moonso) [reply scout/server/blueprints/variants/controllers.py]: Hehe

File: scout/server/blueprints/variants/controllers.py (hunk 8 above)
Comment (moonso) [reply scout/server/blueprints/variants/controllers.py]: Think so

**Comment by moonso** [comment]
Ok that's it, I'll merge now!

**Pull request merged**